celerdata/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "790d8932-0833-43ac-b9d8-d6d0a4f11517",
+  "app_id": "celerdata",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gathers CelerData metrics and logs",
+    "title": "CelerData",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Category::Data Stores",
+      "Category::Log Collection"
```

Comment:
Please include a submitted data type and if applicable, queried data type as mentioned [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags).

Reply:
‌The dashboard and log pipelines have been added, but encountered a strange CI error:
![image](https://github.com/DataDog/integrations-extras/assets/7991096/ab7a2e10-e294-4641-895f-79b4238aebfd)
I added two files: `celerdata.yml` and `celerdata_tests.yaml`

Reply:
found the reason, and fixed it